worksheet.rb

```diff
@@ -1,18 +1,35 @@
 ########################################################
 # Step 1: Establish the layers
 
-# In this section of the file, as a series of comments,
-# create a list of the layers you identify.
-# Which layers are nested in each other?
+# In this section of the file, as a series of comments, create a list of the layers you identify.
+    #  There are three layers: all data for all rides, data for drivers, and data for rides
+    #  Things: driver id, rider id, cost, date, rating, rides
+    #  Relationships: driver id, rides belong to a driver; rider id, cost, date, rating belong to a ride, etc.
```

Comment:
The file is a 2D data structure rows and columns.  The whole file is a list of ride data.  Each row is information about each trip.